### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bezin**
Add hint to Contao Slack Notification Extension

### CODE REVIEW

**Comment by fritzmg** [comment]
There are a few more that can be found via https://extensions.contao.org/?q=notification%20center&pages=1, e.g.

* https://extensions.contao.org/?q=notification%20center&pages=1&p=numero2%2Fcontao-notification_center-aws-sns
* https://extensions.contao.org/?q=notification%20center&pages=1&p=contaoacademy%2Fcontao-zammad-nc-api-bundle
* https://extensions.contao.org/?q=notification%20center&pages=2&p=richardhj%2Fcontao-nc-mailjet-sms
* https://extensions.contao.org/?q=notification%20center&pages=2&p=fenepedia%2Fcontao-klicktipp-gateway

File: README.md [hunk 1]
```diff
@@ -96,7 +96,15 @@ class MyHook
 }
 ```
 
-
+## Community Extensions
+
+* [Slack Gateway](presprog/contao-slack-notification) by [Present Progressive](https://www.presentprogressive.de): Send notifications to Slack channels or users.
```
Comment (aschempp) [inline README.md:101]: the URL looks wrong to me, shouldn't it also point to the extension list?

**Review by aschempp — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (bezin) [reply README.md]: Absolutely, yes, thank you

File: README.md [hunk 2]
```diff
@@ -96,7 +96,15 @@ class MyHook
...
-
+## Community Extensions
+
+* [Slack Gateway](https://extensions.contao.org/?p=presprog%2Fcontao-slack-notification) by [Present Progressive](https://www.presentprogressive.de): Send notifications to Slack channels or users.
+* [AWS SNS Gateway](https://extensions.contao.org/?p=numero2%2Fcontao-notification_center-aws-sns) by [Numero2](https://www.numero2.de): Send notifications via Amazon Simple Notification Services.
+* [Zammad Gateway](https://extensions.contao.org/?p=contaoacademy%2Fcontao-zammad-nc-api-bundle) by [Contao Academy](https://contao-academy.de): Send notifications to Zammad.
+* [KlickTipp Gateway](https://extensions.contao.org/?p=fenepedia%2Fcontao-klicktipp-gateway) by [FenePedia](https://www.fenepedia.de)
+* [Mailjet Gateway](https://extensions.contao.org/?p=mindbird%2Fcontao-notification_center-mailjet) by [mindbird](https://www.mindbird.de)
+* [Mailjet SMS Gateway](https://extensions.contao.org/?p=richardhj%2Fcontao-nc-mailjet-sms) by [Richard Henkenjohan](https://github.com/richardhj)
+* [ClockWork SMS Gateway](https://extensions.contao.org/?p=richardhj%2Fcontao-notification_center_clockworksms) by [Richard Henkenjohan](https://github.com/richardhj)
```
Comment (fritzmg) [inline README.md:107]:
```suggestion
* [ClockWork SMS Gateway](https://extensions.contao.org/?p=richardhj%2Fcontao-notification_center_clockworksms) by [Richard Henkenjohan](https://github.com/richardhj)

```

**Comment by aschempp** [comment]
Thank you @bezin

**Pull request merged**